pkg/storage/SQL/check_if_collaborator.sql

```diff
@@ -0,0 +1,6 @@
+SELECT EXISTS(
+        SELECT 1
+        FROM plan_collaborator
+        WHERE model_plan_id = :model_plan_id
+            AND eua_user_id = :eua_user_id
+    ) AS isCollaborator
```

Comment:
Is the `AS isCollaborator` needed?

Reply:
I tested it, It isn't actually needed, but I do like the clarity it provides in the code. I'd prefer to leave it in, but don't feel too strongly. Thoughts?

Reply:
I'm fine with leaving it! Just wanted to ask as I might assume changing it would break something, but from the looks of it since it's just 1 return value the name might just be for visibility in the SQL statement.